public/javascripts/helpers.js

```diff
@@ -0,0 +1,36 @@
+function ajax(method, url, callback) {
+    var xmlhttp;
+
+    if (window.XMLHttpRequest) {
+        // code for IE7+, Firefox, Chrome, Opera, Safari
+        xmlhttp = new XMLHttpRequest();
+    } else {
+        // code for IE6, IE5
+        xmlhttp = new ActiveXObject("Microsoft.XMLHTTP");
```

Comment:
This was copy-and-pasted from an earlier project that needed support, I can definitely take this out